docker-compose.yml

```
@@ -88,7 +75,6 @@ services:
       - '${REDIS_PORT}:6379'
 
 volumes:
-  php_socket:
   caddy_data:
   caddy_config:
-  db_data:
+  db_data:
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)